components/WeightComponent/RealWeight.cpp

```diff
@@ -1,28 +1,33 @@
-#include "RealWeight.hpp"
+#include "include/RealWeight.hpp"
 
 using namespace Weight;
 
 RealWeight::RealWeight()
 {
-    hx711_t dev = {
-        .dout = GPIO_NUM_19,
```

Comment:
im assuming this was not on an esp32-cam? (on the esp32-cam these are connected internally to the camera header)

Reply:
i believe the finalized pin assignments that we discussed with hardware was gpio 14/2--dont worry too much about it as we only agreed on that today. but i just want to let you know as these might end up being changed!